commons/src/main/java/io/hyscale/commons/utils/ImageDetailsUtil.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import io.hyscale.commons.commands.provider.ImageCommandProvider;
+import io.hyscale.commons.constants.ToolConstants;
+
+@Component
+public class ImageDetailsUtil {
```

Comment:
1. This util is giving broad scope but the scope of the class is image metadata only.
2.ImageCommandProvider does not have to depend on this ...if we can see the normalize separately.